community/kernel/src/main/java/org/neo4j/kernel/impl/api/ConstraintEnforcingEntityOperations.java

```diff
@@ -67,10 +70,10 @@ public ConstraintEnforcingEntityOperations(
     public boolean nodeAddLabel( KernelStatement state, long nodeId, int labelId )
             throws EntityNotFoundException, ConstraintValidationKernelException
     {
-        Iterator<UniquenessConstraint> constraints = schemaReadOperations.constraintsGetForLabel( state, labelId );
+        Iterator<PropertyConstraint> constraints = uniquePropertyConstraints( schemaReadOperations.constraintsGetForLabel( state, labelId ) );
```

Comment:
This two places use `FilteringIterator` over `schemaReadOperations.constraintsGetForLabel`:
https://github.com/neo4j/neo4j/pull/4915/files#diff-0694c9de7c6c3b2738144757b771b751R143
https://github.com/neo4j/neo4j/pull/4915/files#diff-b6dfce9c54e6a4bf3905633eb252f4daR110
so I think it is worth to make separate methods.